packages/w3m.rb

```diff
@@ -0,0 +1,35 @@
+require 'package'
+
+class W3m < Package
+  description 'w3m is a text-based browser, pager and HTML typesetter'
+  homepage 'http://w3m.sourceforge.net/'
+  version '0.5.3'
```

Comment:
`version '0.5.3-38'`?

Reply:
Again the `-38` is part of debian's versioning, upstream is at 0.5.3

Reply:
So what happens when Debian goes from 0.5.3-38 to 0.5.3-45?

Reply:
maybe the version could be `0.5.3-debian-38` would that be okay?

Reply:
Well, if you are using the Debian source, `0.5.3-38` should be fine.